Add a section about SPRAL in the README.md (#372)
jump-dev/Ipopt.jl: README.md

set_attribute(model, "pardisolib", "/full/path/to/libpardiso")
set_attribute(model, "linear_solver", "pardiso")
```

### SPRAL

If you use Ipopt.jl with Julia ≥ v1.9, the linear solver [SPRAL](https://github.com/ralna/spral) is available.

> You can use it setting the `linear_solver` attribute:

Author: ```suggestion
You can use it by setting the `linear_solver` attribute:
```

```julia
using JuMP, Ipopt
model = Model(Ipopt.Optimizer)
set_attribute(model, "linear_solver", "spral")
```

> Note that the following environment variables must be set if you use SPRAL:

Reviewer: before starting Julia? Or can I set them in Julia with `ENV[...] =  ...` before loading Ipopt?

If the latter, it'd be nice to add a test for this.

Author: It should work if we can set them with `ENV[...] = ...` before the `import Ipopt_jll`.

Author: I tested this following code and it's not working:
```julia
ENV["OMP_CANCELLATION"] = "TRUE"
ENV["OMP_NESTED"] = "TRUE"
ENV["OMP_PROC_BIND"] = "TRUE"

using Ipopt, JuMP

# An optimization problem
model = Model(Ipopt.Optimizer)
@variable(model, x)
@objective(model, Min, (x - 2)^2)

# Use the linear solver SPRAL
set_attribute(model, "linear_solver", "spral")
optimize!(model)
```
I need to start Julia with the environment variables already defined:
```
OMP_CANCELLATION=TRUE OMP_NESTED=TRUE OMP_PROC_BIND=TRUE julia
```

> Note that the following environment variables must be set if you use SPRAL:

Reviewer: ```suggestion
Note that the following environment variables must be set before starting Julia:
```
